pkg/endpoint/endpoint.go

```diff
@@ -82,6 +82,12 @@ var (
 	// controller via getCiliumClient and the sync.Once is used to avoid race.
```

Comment:
In the PR title / commit message, should UpdateState --> UpdateStatus?

Reply:
Done